docs/user-guide/timeline.md

```diff
@@ -5,8 +5,9 @@ The Timeline is a [MapStore](https://mapstore.geo-solutions.it/mapstore/#/) tool
 It makes possible to observe the layers' evolution over time, to inspect the layer configuration at a specific time instant (or in a time range) and to view different layer configurations time by time dynamically through animations.<br>
 
 !!! Warning
-    Timeline actually works only with [WMTS-Multidim extension](https://docs.geoserver.org/stable/en/user/community/wmts-multidimensional/index.html) (WMS time in capabilities is not fully supported) and at least **GeoServer 2.14.5**. Anyway to support all the functionality you need **GeoServer 2.15.2**.
+    The Timeline tool currently works only with WMS layers from GeoServer where the [WMTS-Multidim extension](https://docs.geoserver.org/stable/en/user/community/wmts-multidimensional/index.html) is installed (WMS time values in WMS Capabilities is not supported yet). To use the MapStore Timeline at least **GeoServer 2.14.5** is required, but the recommended version is **GeoServer 2.15.2** to have a complete support for all of the features the Timeline tool can provide (e.g. the filter by viewport).
     From now on, the layers that the Timeline can manage will be addressed as *time layers*.
+    From now on, the layers that the Timeline can manage will be addressed as *time layers*. WMS time values in WMS Capabilities is not supported yet.
```

Comment:
From now on, the layers that the Timeline can manage will be addressed as *time layers*.